Application/EdFi.Ods.Api.IntegrationTestHarness/EdFi.Ods.Api.IntegrationTestHarness.csproj

```diff
@@ -67,4 +69,14 @@
   <ItemGroup>
     <DotNetCliToolReference Include="Microsoft.Extensions.SecretManager.Tools" Version="2.0.0" />
   </ItemGroup>
+  <ItemGroup>
+    <!-- 
+    IMPORTANT: Only migration scripts that should be supported here are:
+      * ODS structure (for adding auth views)
+      * Security Data (for adding security metadata) 
+    -->
+    <EmbeddedResource Include="Artifacts\*\Data\Security\*.sql" />
```

Comment:
We only will support Security Data (for manipulating the security metadata for testing), and ODS structure (for creating views).